src/components/AddRepoForm/AddRepoForm.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux'
+import { getRepository } from '../../actions'
+import { Form, InputGroup, FormControl, Button } from 'react-bootstrap';
+import './AddRepoForm.scss'
+
+const AddRepoForm = () => {
```

Comment:
According to naming - we use folder name as component name, this file should be `component.js` or `container.js`, also we add `index.js` which exports component or container